src/JvModel/PrimaryTypeJv.cs

```diff
@@ -70,7 +70,7 @@ public string ModelTypeName
                         result = IsNullable ? "Long" : "long";
                         break;
                     case KnownPrimaryType.Stream:
-                        result = "Flowable<ByteBuffer>";
+                        result = "Flowable<ByteBufferfer>";
```

Comment:
ByteBuffer?

Reply:
Hmm this is wrong in many ways. The path is somehow never hit. Fixing.